src/test/java/ch/zhaw/robopatrol/res/RouteTest.java

```diff
@@ -99,6 +121,123 @@ public void testDelete() {
         assertThat(points, hasSize(0));
     }
 
+    @Test
+    public void testEquals(){
+      Waypoint p1 = point();
+      Waypoint p2 = point();
+
+      assertThat(p1.equals(p2), is(true));
```

Comment:
I agree, but since this method tests the equals method, I want to call it explicitly since we cannot be a 100% sure that hamcrest actually uses the equals method